src/main/java/com/demo/importer/service/StudentService.java

```diff
@@ -0,0 +1,78 @@
+package com.demo.importer.service;
+
+import com.demo.importer.dto.LogDisplayDto;
+import com.demo.importer.dto.StudentAdditionDto;
+import com.demo.importer.dto.StudentDisplayDto;
+import com.demo.importer.dto.LogAddtionDto;
+import com.demo.importer.entity.LogEntity;
+import com.demo.importer.mapstruct.LogMapper;
+import com.demo.importer.repository.LogRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class StudentService {
+
+    @Value("${post.student}")
+    private String ADD_STUDENT_URL;
```

Comment:
Nice work on adding it to app property instead of hard-coding but please read those blogs, you will need to do some changes

Reply:
Okay